use anyhow::Error;

// account.reorderUsernames
pub async fn account_reorder_usernames() -> Result<(), Error> {
    Ok(())
}

// account.toggleUsername
pub async fn account_toggle_username() -> Result<(), Error> {
    Ok(())
}

// bots.reorderUsernames
pub async fn bots_reorder_usernames() -> Result<(), Error> {
    Ok(())
}

// bots.toggleUsername
pub async fn bots_toggle_username() -> Result<(), Error> {
    Ok(())
}

// channels.reorderUsernames
pub async fn channels_reorder_usernames() -> Result<(), Error> {
    Ok(())
}

// channels.toggleUsername
pub async fn channels_toggle_username() -> Result<(), Error> {
    Ok(())
}

// channels.deactivateAllUsernames
pub async fn channels_deactivate_all_usernames() -> Result<(), Error> {
    Ok(())
}


// channels.checkUsername
pub async fn channels_check_username() -> Result<(), Error> {
    Ok(())
}

// channels.updateUsername
pub async fn channels_update_username() -> Result<(), Error> {
    Ok(())
}

// account.updateUsername
pub async fn account_update_username() -> Result<(), Error> {
    Ok(())
}

// account.checkUsername
pub async fn account_check_username() -> Result<(), Error> {
    Ok(())
}

// contacts.resolveUsername
pub async fn contacts_resolve_username() -> Result<(), Error> {
    Ok(())
}